staging/src/k8s.io/metrics/README.md

```diff
@@ -38,9 +38,7 @@ Import Path: `k8s.io/metrics/pkg/apis/custom_metrics`.
 This API allows consumers to access resource metrics (CPU and memory) for
 pods and nodes.
 
-The API is implemented by Heapster
-(https://github.com/kubernetes/heapster) and metrics-server
-(https://github.com/kubernetes-incubator/metrics-server).
+The API is implemented by [Heapster](https://github.com/kubernetes/heapster) and [metrics-server](https://github.com/kubernetes-incubator/metrics-server).
```

Comment:
Heapster project is retired and we shouldn't recommend it anymore. 

Metrics server was moved to https://github.com/kubernetes-sigs/metrics-server